src/drag-drop-polyfill.ts

```diff
@@ -1392,7 +1392,7 @@ module DragDropPolyfill {
 
         const csDragImage = getComputedStyle( dragImage );
         const durationInS = parseFloat( csDragImage.transitionDuration );
-        if (durationInS === NaN || durationInS ====0) {
+        if (isNan(durationInS) || durationInS ===0) {
```

Comment:
Nearly there, but mind the case sensitivity of js.
It must be `isNaN` but currently is `isNan`